multinode_integration_tests/tests/verify_bill_payment_utils/utils.rs

```diff
@@ -41,46 +40,39 @@ use std::thread;
 use std::time::{Duration, Instant, SystemTime};
 use tiny_hderive::bip32::ExtendedPrivKey;
 use web3::transports::Http;
-use web3::types::{Address, Bytes, SignedTransaction, TransactionParameters, TransactionRequest};
+use web3::types::{
+    Address, Bytes, SignedTransaction, TransactionParameters, TransactionReceipt,
+    TransactionRequest,
+};
 use web3::Web3;
 
 pub type StimulateConsumingNodePayments = fn(&mut MASQNodeCluster, &MASQRealNode, &WholesomeConfig);
 
-pub type StartServingNodesAndLetThemPerformReceivablesCheck =
+pub type StartServingNodesAndLetThemActivateTheirAccountancy =
```

Comment:
This name is too long, prefer this `ActivateServingNodesWithAccountancy` or maybe `NodeActivationHandler`. Please change this name, for god's sake, that'